### DISCUSSION TITLE
Plug-Ins Missing from Drop-Down Menu

### DISCUSSION THREAD

**Discussion by sparksandshadows** [apohl79/audiogridder#586] [General]
Why is it that my Valhalla reverb shows up in the "Available Plug-Ins" menu, but not in the plug-in on the other side?
![image](https://user-images.githubusercontent.com/82538597/119448770-9f892f80-bce6-11eb-9302-1d21f95b356e.png)
![image](https://user-images.githubusercontent.com/82538597/119448884-c47da280-bce6-11eb-9843-688cbeaf267e.png)

This is true in both 1.1.1 and 1.2.0-beta3

**Comment by CaBuffalo** [comment]
You try this:

Go to the plugin menu of AG plugin, check "Disable server filter".

You should be able to see it then.

On Mon, May 24, 2021 at 11:22 PM sparksandshadows ***@***.***>
wrote:

> Why is it that my Valhalla reverb shows up in the "Available Plug-Ins"
> menu, but not in the plug-in on the other side?
> [image: image]
> <https://user-images.githubusercontent.com/82538597/119448770-9f892f80-bce6-11eb-9302-1d21f95b356e.png>
> [image: image]
> <https://user-images.githubusercontent.com/82538597/119448884-c47da280-bce6-11eb-9843-688cbeaf267e.png>
>
> This is true in both 1.1.1 and 1.2.0-beta3
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/586>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6B4B6NO7UZG3KA5FC3TPM6Z3ANCNFSM45OS6FVQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by sparksandshadows** [reply]
Aha! That worked! Thank you!
